packages/govuk-frontend-review/src/views/examples/template-custom/index.njk

```diff
@@ -6,7 +6,7 @@
 {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
 {% from "govuk/components/footer/macro.njk" import govukFooter %}
 
-{% extends "govuk/template.njk" %}
+{% extends "layouts/examples-govuk-template.njk" %}
```

Comment:
I just realised that `template-custom` already has a specified `pageTitle` because that's what the example is testing 🤦🏻 I'm going to leave both `template-custom` and `template-default` out of this change as both of them are individually intended to test defaults and customisation coming from our govuk template.